prometheus-to-sd/translator/translator_test.go

```diff
@@ -537,6 +550,17 @@ int_summary_metric_count{label="l2"} 10
 						createIntPoint(50, start, end),
 					},
 				},
+				{
+					Metric: &v3.Metric{
+						Labels:  map[string]string{"quantile": "0.500000",},
+						Type:   "container.googleapis.com/master/testcomponent/float_summary_metric",
+					},
+					MetricKind: "GAUGE",
+					Points: []*v3.Point{
+						// Like all gauges the translater assumes it is an integer, unfortunately :(
```

Comment:
This is a seperate issue that I'll look into in a separate diff.

I've seen this cause some frustrating issues before, outside of summary metric types.